src/Calendar.webmodeler.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, createElement } from "react";
+
+import { Calendar, CalendarProps } from "./components/Calendar";
+import CalendarContainer, { CalendarContainerProps } from "./components/CalendarContainer";
+
+declare function require(name: string): string;
+type VisibilityMap = {
+    [P in keyof CalendarContainerProps]: boolean;
+};
+
+// tslint:disable-next-line class-name
+export class preview extends Component<CalendarContainerProps, {}> {
+    render() {
+        return createElement("div", {},
```

Comment:
Not need for the div, u can return this  createElement(Calendar, this.transformProps(this.props))

Reply:
Not in this case. The `div` is required